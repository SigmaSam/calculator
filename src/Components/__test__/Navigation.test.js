import { BrowserRouter } from 'react-router-dom';
import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer'; // ES6
import Nav from '../Navigation';

describe('Display Navbar', () => {
  it('renders correctly', () => {
    const tree = TestRenderer.create(<BrowserRouter><Nav /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});