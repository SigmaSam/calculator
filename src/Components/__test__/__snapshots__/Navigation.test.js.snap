// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Navbar renders correctly 1`] = `
<div
  className="NavBar"
>
  <h1
    className="Logo"
  >
    Math Magicians
  </h1>
  <p>
    Here you can do Basic Math.
  </p>
  <p>
    For those times where you forget that almost all OS come with an integrated Calculator.
  </p>
  <div
    className="links"
  >
    <a
      className="link"
      href="/home"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="link"
      href="/quote"
      onClick={[Function]}
    >
      Quote of The Day
    </a>
    <a
      className="link"
      href="/app"
      onClick={[Function]}
    >
      Calculator
    </a>
  </div>
</div>
`;
